src/cpp/README.md

```diff
@@ -9,6 +9,20 @@ This section describes how to add gRPC as a dependency to your C++ project.
 In the C++ world, there's no universally accepted standard for managing project dependencies.
 Therefore, gRPC supports several major build systems, which should satisfy most users.
 
+## Supported Platforms
+
+* Supported: These platforms are officially supported. We test our code on these
+platform and have automated continuous integration tests for them.
+
+* Best Effort: We may or may not have continous integration tests for these, but we are fairly confident that gRPC C++ would work on them.
```

Comment:
Do we have platforms for `Best Effort`?

Reply:
just added arm, ios and android, but I do not have version information for them. I guess we can fill it in as we get more information